### DISCUSSION TITLE
催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更

### DISCUSSION THREAD

**Discussion by Fumeng24** [james-curtis/chaoxing-sign-app#19] [General]
催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更

**Comment by shiwu-1** [comment]
这个可以用手机上面吗

**Reply by eric-gitta-moore** [reply]
这个初始的目的就是在单独一个端上跑的，不过挺久没管了我

**Comment by eric-gitta-moore** [comment]
我也想更，😂，但是还有点问题没解决。
- 目前二维码签到不知道有解法没。
- 签到码签到倒是看到别人网站有解法，但是不知道是怎么弄的，有个PPTA什么的类似接口可以直接读取到签到码的具体数值
- 定位这个还不清楚能不能弄

**Reply by eric-gitta-moore** [reply]
要是有任何新发现非常欢迎同步，我愿意赞助继续开发

**Reply by eric-gitta-moore** [reply]
获取签到码的API不知道是不是公开的API。web 上目前没有找到痕迹
可能在历史版本的app里面
可以在豌豆荚找到所有历史版本的app，挨个逆向

**Comment by ka137** [comment]
可以弄一个iOS苹果系统的吗
